app/models/status_edit.rb

```diff
@@ -45,6 +45,10 @@ class PreservedMediaAttachment < ActiveModelSerializers::Model
   delegate :local?, :application, :edited?, :edited_at,
            :discarded?, :visibility, :language, to: :status
 
+  def with_preview_card?
+    false
```

Comment:
There's probably a discussion to be had about whether we should store a snapshot of the link preview when the status gets edited, in case the preview changes over time for the same link, I'm not sure if this is a case we need to solve for now, and displaying the link previews within the version history of the status is a bit much visually I think?